app/assets/javascripts/miq_application.js

```diff
@@ -830,7 +830,9 @@ function miqAjaxAuth(url) {
 
     // TODO API.autorenew is called on (non-login) page load - when?
   })
-  .then(null, function() {
+  .then(function () {
```

Comment:
aah, can you do this in a separate `.then`? Because now failures from `ws_init` would be ignored..